tests/integration/ceph_bucket_notification_test.go

```diff
@@ -30,6 +30,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	httpEndpointService = "http://http-endpoint.default:8080"
```

Comment:
probably better to construct this string from several strings: serviceName, port, namespace
and reuse them in the `CreateHTTPServer()` function